### DISCUSSION TITLE
the greatest PDF links tools.

### DISCUSSION THREAD

**Discussion by Taigan58** [sumatrapdfreader/sumatrapdf#4518] [Ideas]
dear everybody, everyone,

As I m not programming myself, I can only generate the ideas, and hope, expect, wish, would like that somebody get ionvolved and ... sorry to say, prepare if not do the work, sad, to say, ... 

ok, features 1
Thumbnails ide, this is really great to get your thumbnails, but, why don t you add a little featue, with the mouse per exemple using the roulette, add a zoom in/out feature? with high resolution screen, it is very difficult somethimes to read the titles fonts fo a4 pages ...

feature/idea 2
 why not having a feature when you get lot of pdf open to say, SAVE ALL aand perhaps for the names...

features /idea 3
long time ago, adobe rel4? and after uses to allow to capture a complete web  site, until level 1 to ? 4? 5? and generate a single pdf file with these web pages... this is really great

feature /idea 4
like tools ex pdf24 or related, why not extract the links, images, tables? text or ? anything of interest and make separate files/directories...

i know some tools features are existing but integrating all of them ino sumatrapdf can serious help perhaps  adding external tools links too

justs ideas, will be so helpfull and great,

thanks, courage, merci

D

**Comment by GitHubRulesOK** [comment]
1) Thumbnails are Microsoft provided via previewer
![image](https://github.com/user-attachments/assets/6bcc6c0e-f808-4372-91fe-59105eebc23e)

2) Save all open tabs can be done via a shell script see the comment in https://github.com/sumatrapdfreader/sumatrapdf/discussions/4515

3) Adobe caused problems on the web with their software diversification, such that many things once possible are no longer allowed to be exploited.
SumatraPDF is restricted by Windows Trident as to what is allowed web wise so currently you can open one web address after another but manually.

4) PDF24 is a compendium wrapper of other programs like GhostScript plus other PDF reader writers count how many different files it needs to run, in different parts of the system. 
Whilst SumatraPDF is simply one exe for many formats. SumatraPDF can use GhostScript for reading writing and printing PDF (if GS is installed as reader and printer) but you would need to add desired functions via scripting.

**Comment by Taigan58** [comment]
Dear GitHubRulesOK,

Thank you for providing valuable information regarding the use of
thumbnails in SumatraPDF and the limitations imposed by Windows Trident.
Your insights on using a shell script to save all open tabs and the
comparison between SumatraPDF and PDF24 are greatly appreciated.

We acknowledge the challenges caused by Adobe's software and the
restrictions imposed on SumatraPDF for web-related activities. We will
consider your recommendations on using Ghostscript for reading, writing,
and printing PDF files.

Thank you once again for your input and we look forward to further
engagement on GitHub.

Best regards,


Merci Daniel, wildo, need to find ways to do so...

merci and all the best

D

Le mar. 10 sept. 2024 à 14:23, GitHubRulesOK ***@***.***> a
écrit :

>
>    1.
>
>    Thumbnails are Microsoft provide via previewer
>    image.png (view on web)
>    <https://github.com/user-attachments/assets/6bcc6c0e-f808-4372-91fe-59105eebc23e>
>    2.
>
>    Save all open tabs can be done via a shell script see the comment in
>    #4515 <https://github.com/sumatrapdfreader/sumatrapdf/discussions/4515>
>    3.
>
>    Adobe caused probles on the web wiyth their software such that many
>    things once possible are no longer allowed to be exploited.
>    SumatraPDF is restricted by windows trident as to what is allowed web
>    wise so currently you can open one web address after another but manually.
>    4.
>
>    PDF24 is a compendium wrapper of other programs like GhostScript plus
>    other PDF reader writers count how many different files it needs to run
>    whilst SumatraPDF is simply one exe for many formats. sumatraPDF can use
>    Ghostscript for reading writing and printing PDF if its installed but you
>    would need to add desired functions via scripting.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sumatrapdfreader/sumatrapdf/discussions/4518#discussioncomment-10601954>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOHVDS6O5V5FQMFON6YF6BLZV3QFHAVCNFSM6AAAAABN6N5YKOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRQGE4TKNA>
> .
> You are receiving this because you authored the thread.Message ID:
> <sumatrapdfreader/sumatrapdf/repo-discussions/4518/comments/10601954@
> github.com>
>


-- 
Daniel J.-L. Van Mosnenck


Hendrik Serruyslaan, 62/401 B-8400  Oostende Belgium
tel:+352 691 902897

Les informations contenues dans ce message et/ou ses annexes sont réservées
à l'attention et à l'utilisation de leur destinataire et peuvent être
confidentielles. Si vous n'êtes pas destinataire de ce message, vous êtes
informé que vous l'avez reçu par erreur et que toute utilisation en est
interdite. Dans ce cas, vous êtes prié de le détruire et d'en informer
l'émetteur.

The information in this message and/or attachments is intended solely for
the attention and use of the intended recipient and may be confidential. If
you are not the intended recipient of this message, you are hereby notified
that you have received it in error and that any use of it is prohibited. In
such a case, please delete this message and kindly notify the sender
accordingly.

Die Informationen in dieser Mitteilung und / oder Anhänge sind
ausschließlich für die Aufmerksamkeit und die Verwendung des vorgesehenen
Empfängers bestimmt und kann vertrauliche sein.Wenn Sie nicht der
beabsichtigte Empfänger dieser Nachricht sind , werden Sie hiermit darüber
informiert, dass Sie irrtümlich erhalten haben, und dass jede Nutzung ist
untersagt. In einem solchen Fall, löschen Sie bitte diese Meldung und
freundlicherweise entsprechend benachrichtigen Sie den Absender.
